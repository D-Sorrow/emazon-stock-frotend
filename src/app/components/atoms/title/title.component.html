<div class="title">
    <h1>AÃ±adir categoria</h1>
</div>
