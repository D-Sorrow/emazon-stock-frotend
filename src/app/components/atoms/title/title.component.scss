@import "../../../shared/tokens/typografhy";


.title{

    text-align: center;
    margin: 16px 0px;
    
    h1{

        font-weight: bold;
    }
}
