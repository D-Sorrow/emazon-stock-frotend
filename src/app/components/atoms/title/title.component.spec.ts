import { ComponentFixture, TestBed } from '@angular/core/testing';

import { TitleComponent } from './title.component';

describe('TitleComponent', () => {
  let component: TitleComponent;
  let fixture: ComponentFixture<TitleComponent>;
  let compiled: HTMLElement;

  beforeEach(async () => {
    await TestBed.configureTestingModule({
      declarations: [ TitleComponent ]
    })
    .compileComponents();

    fixture = TestBed.createComponent(TitleComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('Match with the title component', () => {
    expect(compiled).toMatchSnapshot();  
  });

});
