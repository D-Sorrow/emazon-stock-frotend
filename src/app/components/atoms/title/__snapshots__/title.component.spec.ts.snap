// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleComponent Match with the title component 1`] = `
<div
  id="root1"
>
  <div
    class="title"
  >
    <h1>
      
    </h1>
  </div>
</div>
`;
